ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/svarinn/MimeTypeExtensionMapper.java

```diff
@@ -0,0 +1,35 @@
+package no.difi.meldingsutveksling.ks.svarinn;
+
+import com.google.common.collect.Maps;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.MediaType;
+
+import java.util.HashMap;
+
+public class MimeTypeExtensionMapper {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1118)